lib/good-influx-udp.js

```diff
@@ -12,6 +12,9 @@ const internals = {
 
 class GoodInfluxUdp extends GoodUdp {
     constructor(endpoint, config) {
+        if (isNaN(config.threshold) || config.threshold > 5) {
```

Comment:
Oops!  Not sure how that got lost in the shuffle.  Will reinstate when I get home in a few hours.